engine.js

```diff
@@ -19,12 +21,33 @@ module.exports = function (options) {
   var types = options.types;
 
   var length = longest(Object.keys(types)).length + 1;
-  var choices = map(types, function (type, key) {
+  var typeChoices = map(types, function (type, key) {
     return {
       name: rightPad(key + ':', length) + ' ' + type.description,
       value: key
     };
   });
+  var scopePrompt = {
+    type: 'input',
+    name: 'scope',
+    message: 'What is the scope of this change (e.g. component or file name)? (press enter to skip)\n'
+  };
+  try {
+    var choices = JSON.parse(
+      fs.readFileSync(
+        path.join(path.resolve(''), 'package.json'),
+        'utf8'
+      )
+    ).config.commitizen.scopeList;
+    if (choices) {
+      scopePrompt = {
+        type: 'checkbox',
+        name: 'scope',
+        message: 'Select the scope(s) of this change?',
+        choices: choices
+      };
+    }
+  } catch (error) {}
```

Comment:
I think don't think we should choke the error. We should either do something with it or remove the try/catch and let it crash and report the error.